pyproject.toml

```diff
@@ -13,7 +13,7 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-django = ">=3,<5"
+django = "^4.2.11"
```

Comment:
Do we need to change this? or is it enough to just update lock file as in the dependabot PR? Most services are migrated to 4.2 now, but some using the package may only be on an earlier version of 4.2. Maybe `>=4.2` is enough

Reply:
Yep happy to just update lock file, seems a better approach 👍🏻